app/views/planning_applications/review/local_policies/_form.html.erb

```diff
@@ -3,51 +3,19 @@
-        ) do %>
-      <%= form.govuk_check_boxes_fieldset :areas, multiple: true, legend: {text: "Select the topics you want to comment on", size: "m"} do %>
-        <%= form.fields_for :local_policy_areas, form.object.owner.local_policy_areas do |ff| %>
-          <%= ff.govuk_check_box :areas, :"#{ff.object.area}", label: {text: ff.object.area.to_s.humanize.to_s}, checked: @local_policy.local_policy_areas.any? { |local_policy_area| local_policy_area.area == ff.object.area.to_s }, disabled: !editable do %>
-            <%= ff.hidden_field :area, value: ff.object.area.to_s %>
-            <%= ff.hidden_field :id, value: ff.object.id %>
-            <%= ff.govuk_text_field :policies, label: {text: "Which policies are relevant", size: "s"}, hint: {text: "Enter the policy reference"} %>
-            <%= ff.govuk_radio_buttons_fieldset(:policy, legend: {size: "s", text: "Is there relevant guidance?"}) do %>
-              <%= ff.govuk_radio_button :policy, "Yes", label: {text: "Yes"}, checked: ff.object.guidance.present? do %>
-                <%= ff.govuk_text_field :guidance, label: {text: "Which guidance? (e.g. the design code)", size: "s"} %>
-              <% end %>
-              <%= ff.govuk_radio_button :policy, "No", label: {text: "No"}, checked: ff.object.id.present? && ff.object.guidance.blank? %>
-            <% end %>
-            <%= ff.govuk_text_area :assessment, label: {text: "Enter your assessment", size: "s"} %>
-          <% end %>
-        <% end %>
-      <% end %>
-    <% end %>
-
     <%= form.govuk_radio_button(
-          :action, "rejected", disabled: !editable, checked: !@review_local_policy.status == "accepted" && @review_local_policy.comment?, label: {text: "Return to officer with comment"}
+          :action, "rejected", checked: @review_local_policy.action == "rejected", label: {text: t(".no_label")}
         ) do %>
-      <%= form.govuk_text_area(
-            :comment, readonly: !editable, rows: 6, label: {text: "Comment"}
-          ) %>
+      <%= form.govuk_text_area(:comment, rows: 6, label: {text: t(".comment_label")}) %>
     <% end %>
   <% end %>
 
-  <div class="govuk-button-group">
-    <% if editable %>
-      <%= form.submit "Save and mark as complete", class: "govuk-button", data: {module: "govuk-button"} %>
-      <%= form.submit "Save and come back later", class: "govuk-button govuk-button--secondary", data: {module: "govuk-button"} %>
-    <% end %>
-
-    <%= back_link %>
-
-    <% if !editable %>
-      <%= govuk_link_to "Edit review policy guidance", edit_planning_application_review_local_policy_path(@planning_application, @review_local_policy) %>
-    <% end %>
-  </div>
+  <%= form.govuk_submit("Save and mark as complete") do %>
+    <%= form.govuk_submit("Save and come back later", secondary: true) %>
+    <%= govuk_button_link_to("Back", planning_application_review_tasks_path(@planning_application), secondary: true) %>
+  <% end %>
```

Comment:
Is there a reason you're getting rid of the "edit" link if the review has been filled out?

Reply:
Now there is an edit link to go to the edit page directly